libs/shared/console-shared/src/lib/job-general-setting/job-general-setting.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import { render } from '__tests__/utils/setup-jest'
+import { wrapWithReactHookForm } from '__tests__/utils/wrap-with-react-hook-form'
+import { BuildModeEnum, GitProviderEnum } from 'qovery-typescript-axios'
+import { ServiceTypeEnum } from '@qovery/shared/enums'
+import { JobGeneralData } from '@qovery/shared/interfaces'
+import JobGeneralSetting from './job-general-setting'
+
+describe('JobGeneralSetting', () => {
```

Comment:
The test is light because you didn't test condition when we display fields or not, can you add more test?